pkg/projectfile/projectfile.go

```diff
@@ -1446,3 +1447,10 @@ func CleanProjectMapping(cfg ConfigGetter) {
 		logging.Debug("Could not clean project mapping in config, error: %v", err)
 	}
 }
+
+func SetRecentlyUsedNamespace(cfg ConfigGetter, namespace string) {
+	err := cfg.Set(constants.LastUsedNamespacePrefname, namespace)
+	if err != nil {
+		logging.Debug("Could not set recently used namespace in config, error: %v", err)
```

Comment:
I think we normally use `multilog.Error()` for these sorts of things, but I don't know if that's needed here.

Reply:
Opted to not use it here as we are now allowing this argument to be either a valid namespace or just a project name.